styles.css

```diff
@@ -16,6 +16,11 @@ body {
   margin: 0;
 }
 
+main {
+  display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` then assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.

Reply:
Hi @AdnanOlarmmi, I have fixed these issues.